src/go/cmd/token-vendor/api/v1/v1.go

```diff
@@ -255,6 +255,45 @@ func (h *HandlerContext) tokenOAuth2Handler(w http.ResponseWriter, r *http.Reque
 	w.Write(tokenBytes)
 }
 
+// Handle requests to verify if a given JWT is valid
+//
+// This handler provides a backend for common proxy servers forward authentication.
```

Comment:
I feel this will be confusing to future readers who don't have the context you have now, especially the noun phrase "common proxy servers forward authentication". Does that refer to the nginx auth-url webhook? Or to another proxy?

Should we just copy-paste the docs from above and avoid documenting how this is used here? That is to say,  #copying:

```
// The robot identifies itself using its private key to sign a JWT. The JWT is
// verified by the token vendor using the public key from the key repository and
// the device identifier from the `iss` key from the JWT's payload section.
```

And explain that this can be used as an authentication check but that it doesn't apply authz beyond comparing to the list of keys in the key repository?

Reply:
I replaced it by a copy+paste from the readme, since that seems like an reasonable explanation to me.